plugin/clone/src/clone_plugin.cc

```diff
@@ -70,6 +70,10 @@ uint clone_max_io_bandwidth;
 /** Clone system variable: If network compression is enabled */
 bool clone_enable_compression;
 
+unsigned long clone_compression_algorithm;
```

Comment:
```suggestion
ulong clone_compression_algorithm;
```
for symmetry with `uint`